src/main/java/by/itechart/api/controller/UserController.java

```diff
@@ -33,10 +33,15 @@
     }
 
     @PatchMapping("{id}")
-    public ResponseEntity<UserDTO> updateUser(@PathVariable Long id, @RequestBody UserDTO user) {
+    public ResponseEntity<UserDTO> updateUserPartially(@PathVariable Long id, @RequestBody UserDTO user) {
```

Comment:
As I remember we agreed to create a new model class for user update data. Are you going to add it as a next step?

Reply:
Do you mean new DTO or I misunderstood?

Reply:
Yes, a new DTO class, e.g. `UserUpdateInputDTO` or so